docs/data-management/mriqc.md

```diff
@@ -10,10 +10,96 @@
+
+!!! warning "Immediately report images deemed *exclude*, as an issue in the dataset's repository"
+
+### Assessing functional images
+
+!!! danger "Insufficient quality of an fMRI run requires recalling the session"
+
+    - [ ] Immediately report images deemed *exclude*, as an issue in the dataset's repository.
+    - [ ] Proceed to scheduling an extra session after the initially-planned scanning period.
+
+#### RSfMRI
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
+- [ ] Scroll down to the standard-deviation (std) mosaic and apply the [exclusion criteria](qaqc-criteria.md#standard-deviation-mosaic)
+- [ ] Scroll down to the background noise mosaic and apply the [exclusion criteria](qaqc-criteria.md#background-noise-mosaic)
+- [ ] Scroll down to the fMRI summary plot and apply the [exclusion criteria](qaqc-criteria.md#fmri-summary-plot) as well as [the exclusion criteria specific to the BHT](qaqc-criteria.md#resting-state).
+
+#### QCT
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
+- [ ] Scroll down to the standard-deviation (std) mosaic and apply the [exclusion criteria](qaqc-criteria.md#standard-deviation-mosaic)
+- [ ] Scroll down to the background noise mosaic and apply the [exclusion criteria](qaqc-criteria.md#background-noise-mosaic)
+- [ ] Scroll down to the fMRI summary plot and apply the [exclusion criteria](qaqc-criteria.md#fmri-summary-plot) as well as [the exclusion criteria specific to the BHT](qaqc-criteria.md#quality-control-task) 
+
+#### BHT
+- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the first mosaic (BOLD average) and apply the [exclusion criteria](qaqc-criteria.md#bold-average)
+- [ ] Scroll down to the standard-deviation (std) mosaic and apply the [exclusion criteria](qaqc-criteria.md#standard-deviation-mosaic)
+- [ ] Scroll down to the background noise mosaic and apply the [exclusion criteria](qaqc-criteria.md#background-noise-mosaic)
+- [ ] Scroll down to the fMRI summary plot and apply the [exclusion criteria](qaqc-criteria.md#fmri-summary-plot) as well as [the exclusion criteria specific to the BHT](qaqc-criteria.md#breath-holding-task) 
+
+## Visualizing *MRIQC*'s group reports
+
+### Assessing functional images
+- [ ] Open the *MRIQC* group report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Visualize the IQMs distributions and apply the [exclusion critera](qaqc-criteria.md#group-report) for all BOLD scans and apply [these exclusion criteria](qaqc-criteria.md#group-report-1) to RSfMRI.
+
+### Assessing diffusion images
+
+
+!!! danger "Insufficient quality of a dMRI run requires recalling the session"
```

Comment:
Not sure we want to do this, I think we can drop dMRI sessions without collecting a full session again.

Reply:
I was thinking the point was to be able to compute FC and SC for each session.
As such, why do we recollect a session in case of RSfMRI exclusion but not dMRI?

Reply:
Statistical power

Reply:
So we expect effect sizes to be smaller in FC than in SC?

Reply:
Let's move this to an issue, to be able to close this PR.